web-module/src/main/webapp/templates/celAjax/struct/Tiny4Config.vm

```diff
@@ -129,52 +129,78 @@ $!services.rteconfig.setRteConfigHint("tinymce4")
 #set($filebaseURL = "$!{filebaseURL}&src_doc=${filePickerSpace}.${filePickerDoc}&columns=10")
 #set($filebaseURL = "$!{filebaseURL}&root=${filePickerSpace}&hasUpload=$hasUpload")
 ##
+## lookup the rte_styles field in the XWiki and WebPreference
+#set($rteStyles = $services.rteconfig.getRTEConfigField('styles'))
+## get multilingual style names
+#set($mlRteStyleArray = [])
+#set($mlRteStyleName = '')
+#foreach($rteStyleElement in $rteStyles.split('[=;]'))
+  #if($velocityCount%2 == 1)
```

Comment:
somewhat brittle implementation, an incomplete style definition may mess up everything, e.g.:
`dictkey1,dictkey2=cssClass2`
`dictkey1=cssClass1,,dictkey2=cssClass2`
`,dictkey1=cssClass1,dictkey2=cssClass2`
it may be better to split twice, first by `,`/`;` and second by `=`